README.md

```diff
@@ -16,24 +16,26 @@ With [Fisher](https://github.com/jorgebucaran/fisher):
 $ fisher install acomagu/fish-async-prompt
 ```
 
-If your prompt doesn't work correctly, try changing the configuration.
```

Comment:
Why is it not necessary? I mean, I want the user to tune the configurations if they have problems.